### ISSUE TITLE
Loading an application from other domain

### ISSUE THREAD

**Issue by reyryan** [PlaceMe-SAS/single-spa-angular-cli-examples#15]
Is it possible to load an Angular 4 application deployed from a different domain?

Also, I am planning to create a portal application which can call other Angular applications served from different locations, is this possible ?

TIA.

**Comment by robinComa** [comment]
Yes, only your index.html is load by ajax (no problem for css and js files) you have to enable CORS on your app server side.

**Comment by reyryan** [comment]
Hello, thank you for your immediate response.

Do you have a sample application on that, please ?
My main problem now is the routing configuration part and how I can pass around the session data or token across applications.

Thanks again.

**Comment by robinComa** [comment]
No exemple but CORS is a standard.
If you don’t use the same domain you cannot local storage or cookie session... maybe you can use oauth with token on all you url? Or in memory session (if you reload the page you have te reauth the user)
